### PR TITLE
Created new release notes for MTC 1.7.8

### PR DESCRIPTION
**Pull request by HagayVider1**
GH# 
OCPBUGS# [MIG-1316](https://issues.redhat.com//browse/MIG-1316)
OSDOCS# [MIG-1316](https://issues.redhat.com//browse/MIG-1316)

Version(s):

OpenShift Container Platform versions 4.10, 4.11, 4.12
MTC version 1.7.8

Issue:

[MIG-1316](https://issues.redhat.com//browse/MIG-1316): MTC 1.7.8 Release Notes
https://issues.redhat.com/browse/MIG-1316

Link to docs preview:

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/migration-mtc-release-notes-1-7-8.adoc [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+= {mtc-full} 1.7.8 release notes
+
+[id="new-features-and-enhancements-1-7-8_{context}"]
+== New features and enhancements
+
+This release has the following new features and enhancements:
+
+* Added configuration option to set 'spc_t' as SELinuxOptions Type in Rsync containers.
+
+* Added configuration option to override Velero images.
```
Comment (stillalearner) [inline modules/migration-mtc-release-notes-1-7-8.adoc:16]: @HagayVider1 This should be in the Resolved issues section.

"Fixed configuration option to override Velero image."

Comment (HagayVider1) [reply modules/migration-mtc-release-notes-1-7-8.adoc]: @stillalearner 
Resolved. Please check.

**Review by stillalearner — APPROVED** [review]
lgtm

**Pull request closed without merging**